pydicom/jsonrep.py

```diff
@@ -41,3 +47,202 @@ def convert_to_python_number(value, vr):
         else:
             value = number_type(value)
     return value
+
+
+class JsonDataElementConverter(object):
```

Comment:
Can you add this to the API docs please? Maybe create a new section.

Reply:
Actually, this is sort of a private class - not sure how we handle these...

Reply:
Whatever you decide is fine.